aiida/cmdline/commands/work.py

```diff
@@ -8,12 +8,19 @@
 # For further information please visit http://www.aiida.net               #
 ###########################################################################
 import click
+import logging
 from tabulate import tabulate
 from aiida.cmdline.commands import work, verdi
 from aiida.cmdline.baseclass import VerdiCommandWithSubcommands
 
 CONTEXT_SETTINGS = dict(help_option_names=['-h', '--help'])
 
+LOG_LEVELS = [
```

Comment:
I think the following might be a little bit more robust:

    LOG_LEVELS = [level_name for level, level_name in logging._levelNames.iteritems() if isinstance(level, int)]

That will produce a sorted list of the registered level names from least severe to most severe.
What do you think?

Other than this everything looks great